armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -759,12 +796,10 @@ EOF
 
 # include Wifi credentials, if specified
 if [[ -n "${BASE_WIFI_SSID}" ]]; then
-  cat << EOF > /etc/network/interfaces.d/wlan0.conf
-auto wlan0
-iface wlan0 inet dhcp
-  wpa-ssid ${BASE_WIFI_SSID}
-  wpa-psk ${BASE_WIFI_PASS}
-EOF
+  sed -i '/WPA-SSID/Ic\  wpa-ssid ${BASE_WIFI_SSID}' /opt/shift/config/wifi/wlan0.conf
```

Comment:
Shouldn't we be replacing the lower-case `wpa-ssid` here? (Not `WPA-SSID`.)

(And elsewhere.)

Reply:
I consistently `sed` for case-insensitive "UPPER CASE" search strings, also in other scripts. This way, if it works, I'm sure it's really not case-sensitive.